### PR TITLE
enable all tests on macOS

### PR DESCRIPTION
**Pull request by Theodus**

### CODE REVIEW

**Comment by SeanTAllen** [comment]
@Theodus is it looking for 3 release versions?

**Comment by SeanTAllen** [comment]
@Theodus can you bump this to try it running again?

**Comment by SeanTAllen** [comment]
@Theodus can you rebase this against master and force push so we can see where we are with this?

File: test/main.pony [hunk 1]
```diff
@@ -70,7 +66,7 @@ class _TestSync is UnitTest
 
 class _TestSelect is UnitTest
   let _ponyc_versions: Array[String] val =
-    ["release-0.33.1"; "release-0.33.0"]
+    ["release-0.33.2"; "release-0.34.0"]
```
Comment (SeanTAllen) [inline test/main.pony:69]: Do we have to update this for every release?

Comment (Theodus) [reply test/main.pony:69]: No, this test just requires 2 versions available on all supported platforms

Comment (SeanTAllen) [reply test/main.pony:69]: So what are the hardcoded versions? I'm confused by those.

Comment (Theodus) [reply test/main.pony:69]: The test installs both versions, and tests selecting between them using ponyup.

Comment (SeanTAllen) [reply test/main.pony:69]: Right so if we add FreeBSD support, then we need to change these right? And it impacts on every platform.

Comment (Theodus) [reply test/main.pony:69]: Yes, and the tests would require two releases (or nightly releases temporarily) to be available.

**Pull request merged**